ijson: not packaged in debian
serde_json: throws lots of compile errors in src/impls/serde_json.rs
nightly: incompatible with release build

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,10 +33,2 @@ repository = "https://github.com/timothe
 
-[dependencies.ijson]
-version = "^0.1"
-optional = true
-
-[dependencies.serde_json]
-version = "^1.0.71"
-optional = true
-
 [dependencies.slab]
@@ -53,4 +45,2 @@ all-impls = [
     "smallvec",
-    "serde_json",
-    "ijson",
 ]
@@ -61,3 +51,2 @@ default = [
 ]
-nightly = []
 std = []
